app/uk/gov/hmrc/helptosave/controllers/HelpToSaveAuth.scala

```diff
@@ -58,15 +58,15 @@ class HelpToSaveAuth(htsAuthConnector: AuthConnector) extends BaseController wit
   def handleFailure(): PartialFunction[Throwable, Result] = {
     case _: NoActiveSession ⇒
       logger.warn("user is not logged in, probably a hack?")
-      Forbidden("no active session found for logged in user")
+      Unauthorized
 
-    case _: InsufficientConfidenceLevel | _: InsufficientEnrolments ⇒
-      logger.warn("unexpected: not met required ConfidenceLevel for logged in user")
-      Forbidden("not met required ConfidenceLevel for logged in user")
+    case e: InternalError ⇒
```

Comment:
what happens in this case ? `_: InsufficientConfidenceLevel | _: InsufficientEnrolments `

Reply:
those will get picked up in the `AuthorisationException` case